File: Question 09/palindrome.java
class Solution {
  public void answer(Object input) {

         String str = (String) input;
         String org = str;
	     String rev="";
	     
	     int len=str.length();
	     
	     for(int i=len-1; i>=0; i--)
	     {
	    	 rev = rev+str.charAt(i);
	     }
	     if(org.equals(rev))
	     {
	    	 System.out.println("Palindrome");
	     }
	     else
	     {
	    	 System.out.println("Not Palindrome");
	     }
  }
}

File: Question 09/palindrome.js
//write a program to find wether a string is palindrome or not.
function answer(str) {
    let org = str;
    let rev = "";

    for (let i = str.length - 1; i &gt;= 0; i--) {
        rev += str.charAt(i);
    }

    if (org === rev) {
        console.log("Palindrome");
    } else {
        console.log("Not Palindrome");
    }}
